deployments/hadoop-yarn/ansible/27-install-zeppelin.yml

```
@@ -221,7 +221,7 @@
 
     - name: "Download and install Zeppelin"
       unarchive:
-        src: "https://downloads.apache.org/zeppelin/zeppelin-{{zepvers}}/{{zepname}}.tgz"
+        src: "{{zepurl}}"
```

`zepurl` is too short to be meaningful. We already have a variable called `zeppelinurl` in our client toolkit scripts, which is the URL for the Zeppelin service endpoint. 

This is the URL of the Zeppelin tar file that we want to download, so the name should be descriptive of that to avoid confusion.

```suggestion
        src: "{{zeptarurl}}"
```

We have plenty of electrons, so three more characters isn't going to cause a problem.